lib/api.php

```diff
@@ -185,6 +191,20 @@ public function log($fields);
     public function guid();
 
     /**
+     * Returns the unique identifier for the trace of which this span is a part.
+     *
+     * @return string
+     */
+    public function traceGUID();
+
+    /**
+     * Sets the GUID of the containing trace onto this span.
+     *
+     * @param string $traceGUID
+     */
+    public function setTraceGUID($traceGUID);
```

Comment:
These all seemed to be used by `ClientTracer`, regardless of whether a `NoOpSpan` or a `ClientSpan` was provided. I added them to make the interface consistent.